### ISSUE TITLE
Weird behaviour in the inContext editor

### ISSUE THREAD

**Issue by einargudnig** [locize/locize#167]
## 🐛 Bug Report

When I use the inContext Editor to edit my translation the changes are not visible in the browser.
In one instance I can press a button (show an answer to a FAQ question (visible in the video)) and then changes are visible.
Everything else seems to be working.

Video explaining the issues https://www.loom.com/share/a2f7668972ee44b4bc0d496067537ca5

## To Reproduce

Video explaining the issues https://www.loom.com/share/a2f7668972ee44b4bc0d496067537ca5
My coworker had the same issue. I'm not sure how I reproduce it differently.

## Expected behavior

When I update a string in inContext editor and press 'Save' I expect the change to be visible in the inContext editor. It was working like that the last time I used it, couple of weeks ago.

I expect it to behave the same way it does here: https://youtu.be/f0ukRI0LMfo?t=292 (youtube video from your youtube channel)

## Your Environment

- *runtime version*: Google chrome, Arc browser
- *module version*: >=19.0.0
- *os*: Mac, Windows
- *any other relevant information*

**Comment by jamuhl** [comment]
I assume you're following the readme: https://github.com/locize/locize#with-i18next

- using the plugin
- bind the extra event

if so...is it possible that the (looks like react) react components are not using `useTranslation` hook or `withTranslation` HOC? https://react.i18next.com/latest/usetranslation-hook <- this is basically what triggers a rerender on the component on receiving a `editorSaved` event from the locize script

**Comment by einargudnig** [comment]
Thanks for a quick answer!
I took a look at this an noticed we had not added the binding for the extra event.
- We have the locize plugin our init file.
- We are using the `useTranslation` to trigger the translation in these component so it seems like should work.

I added the binding to our i18n init file but it was the same.

I will look further into at and try to figure out more details. To the best of my knowledge it was working "normally" before, and nothing changed.
